modules/exploits/unix/http/schneider_electric_net55xx_encoder.rb

```diff
@@ -0,0 +1,188 @@
+﻿##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'net/ssh/command_stream'
```

Comment:
You shouldn't need this if you include the mixin below.

Reply:
Resolved at commit [6f21abf](https://github.com/rapid7/metasploit-framework/pull/12084/commits/6f21abfe08bde97fbf2223f390b1e288c5da5721)